### ISSUE TITLE
False positive when running update_build_files

### ISSUE THREAD

**Issue by dirtyhabits97** [cgrindel/rules_swift_package_manager#1021]
Just ran into this very particular case, where running `bazel run //:update_build_files` results in some warnings:
```
gazelle: Unable to find dependency labels for s, sCount, sFormat, sOffset, s_count, s_format, s_offset
gazelle: Unable to find dependency labels for MachO, s, sCount, sFormat, sOffset, s_count, s_format, s_offset
gazelle: Unable to find dependency labels for s
```

<img width="884" alt="Screenshot 2024-04-17 at 19 58 05" src="https://github.com/cgrindel/rules_swift_package_manager/assets/25568998/b7375472-a292-45fd-b9bf-b7f6331785d4">

This is because some of the variable names in my project start with `import`. Example:
<img width="841" alt="Screenshot 2024-04-17 at 19 59 31" src="https://github.com/cgrindel/rules_swift_package_manager/assets/25568998/112921c6-c892-4d26-9718-749450655237">

I assume the command tries to look for the dependency `sFormat` because it splits `importsFormat` into `import sFormat`.

Not that big of a deal, since the command returns a successful exit code and this is an uncommon scenario
